api/health.go

```diff
@@ -68,7 +73,19 @@ func (d *HealthCheckDefinition) MarshalJSON() ([]byte, error) {
 		Timeout:                        d.Timeout.String(),
 		DeregisterCriticalServiceAfter: d.DeregisterCriticalServiceAfter.String(),
 		Alias:                          (*Alias)(d),
-	})
+	}
+
+	if d.IntervalDuration != 0 {
+		out.Interval = d.IntervalDuration.String()
+	}
```

Comment:
Should probably also have:

```
} else if d.Interval != 0 {
   out.Interval = d.Interval.String()
}```

Same thing goes for Timeout and DeregisterCritcalServiceAfter